packages/py-api/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.11 as build
+
+# For some reason when installing poetry with curl, after that poetry is not found
```

Comment:
Because you need to add it to the PATH

Reply:
but if pip yields the same result, there is no need to play around with using curl